src/EfuncMsg.chpl

```diff
@@ -42,6 +42,100 @@ module EfuncMsg
       :throws: `UndefinedSymbolError(name)`
       */
 
+
+//  This section is a rewrite of trig and hyp functions in new interface.
+//  This comment will be updated as other functions are rewritten, and deleted
+//  once the rewrite is complete.
+
+//  Error-throwing versions of each function are indented below their "parents"
+//  in order to improve readability.
+
+    @arkouda.registerCommand (name="sin")
+    proc sine (x : [?d] ?t) : [d] real throws
+        where (t==int || t==real || t==uint) { return sin(x); }
+       
+        proc sine (x : [?d] ?t) : [d] real throws
+            { throw new Error ("Invalid data type for sin"); }
+
+    @arkouda.registerCommand (name="cos")
+    proc cosine (x : [?d] ?t) : [d] real throws
+        where (t==int || t==real || t==uint) { return cos(x); }
+       
+        proc cosine (x : [?d] ?t) : [d] real throws
+            { throw new Error ("Invalid data type for cos"); }
```

Comment:
If you want you can give a slightly more informative error message using string formatting:
`"cosine does not support type %s".format(type2str(t))`

Reply:
That is better, yes.  I think it's OBE by the python checking, but I'll change it just in case.